provider/cmd/pulumi-resource-hcloud/schema.json

```diff
@@ -4132,7 +4132,7 @@
                 },
                 "autoDelete": {
                     "type": "boolean",
-                    "description": "Whether auto delete is enabled.\n"
+                    "description": "Whether auto delete is enabled.\n`Important note:`It is recommended to set `auto_delete` to `false`, because if a server assigned to the managed ip is getting deleted, it will also delete the primary IP which will break the TF state.\n"
```

Comment:
we probably want to remove the `TF` here, no? not sure on the nest way to do this :/